packages/alert/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@availity/mui-alert",
+  "version": "0.0.0",
+  "description": "Availity MUI Alert Component - part of the @availity/element design system",
+  "keywords": [
+    "react",
+    "typescript",
+    "availity",
+    "alert",
+    "mui"
+  ],
+  "homepage": "https://availity.github.io/element/?path=/docs/components-alert-introduction--docs",
+  "bugs": {
+    "url": "https://github.com/Availity/element/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Availity/element.git",
+    "directory": "packages/alert"
+  },
+  "license": "MIT",
+  "author": "Roxanne Young <roxanne.young@availity.com>",
+  "browser": "./dist/index.js",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    }
+  },
+  "scripts": {
+    "build": "tsup src/index.ts --format esm,cjs --dts",
+    "dev": "tsup src/index.ts --format esm,cjs --watch --dts",
+    "clean": "rm -rf dist",
+    "clean:nm": "rm -rf node_modules",
+    "bundlesize": "bundlesize",
+    "publish": "yarn npm publish --tolerate-republish --access public",
+    "publish:canary": "yarn npm publish --access public --tag canary"
+  },
+  "devDependencies": {
+    "@mui/material": "^5.11.9",
```

Comment:
I think we should set `@mui/material` as a dependency rather than a peer. I want to get away from the peerDependency way we did with `reactstrap`

Reply:
added material dependency to `element` instead, left peer dependency for components.